README.md

```diff
@@ -3,3 +3,20 @@
 The public website for the FLIF project.
 
 * **[FLIF.info](http://flif.info)**
+
+* * *
+
+## Contributing
+
+To run the app locally:
```

Comment:
OMG, yes :/ this definitely is overcomplicating the project (site) for very little benefit IMO.